frontend/src/components/dashboard/visitors/ButtonComponent.vue

```diff
@@ -0,0 +1,95 @@
+<template>
+    <div
+        class="button-card bg-white d-inline-flex justify-content-start align-items-center"
+    >
+        <div
+            class="card-image"
+        >
+            <img
+                class="button-icon"
+                :src="iconSrs"
+            >
+        </div>
+        <div
+            class="card-text"
+        >
+            <div
+                class="character-text text-no-wrap headline"
+            >
+                {{ character }}
+            </div>
+            <div
+                class="title-text text-no-wrap caption"
+            >
+                {{ title }}
+            </div>
+        </div>
+    </div>
+</template>
+
+<script>
+    import getIcon from "./iconsConfig"
+
+    export default {
+        name: 'ButtonComponent',
+        props: {
+            title: {
+                type:String,
+                required: true
+            },
+            character: {
+                type:String,
+                required: true
+            },
+            iconKey: {
+                type:String,
+                required: true
+            }
+        },
+        data () {
+            return {
+                iconSrs: ''
+            }
+        },
+        created () {
+            this.iconSrs = "data:image/png;base64," + getIcon(this.iconKey);
```

Comment:
it's better don't define images like that, let webpack do all optimization work

better create actual files in `assets/icons`, pass prop that will define icon name (key key that will be transpiled to icon name), this line better to remove